_posts/2023-07-17-13.0.0-release.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: "Apache Arrow 13.0.0 Release"
+date: "2023-07-17 00:00:00"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+
+The Apache Arrow team is pleased to announce the 13.0.0 release. This covers
+over 3 months of development work and includes [**XXX resolved issues**][1]
+from [**YYY distinct contributors**][2]. See the [Install Page](https://arrow.apache.org/install/)
+to learn how to get the libraries for your platform.
+
+The release notes below are not exhaustive and only expose selected highlights
+of the release. Many other bugfixes and improvements have been made: we refer
+you to the [complete changelog][3].
+
+## Community
+
+Since the 12.0.0 release, Marco Neumann, Gang Wu, Mehmet Ozan Kabak and Kevin Gurney
+have been invited to be committers.
+Matt Topol, Jie Wen, Ben Baumgold and Dewey Dunnington have joined the
+Project Management Committee (PMC).
+
+Thanks for your contributions and participation in the project!
+
+## Columnar Format Notes
+
+## Arrow Flight RPC notes
+
+## C++ notes
+
+## C# notes
```

Comment:
@eerhardt could you help with the C# notes for 13.0.0 ?

Reply:
```suggestion
## C# notes

### Enhancements

* The [C Data Interface](https://arrow.apache.org/docs/format/CDataInterface.html) is now supported in the .NET Apache.Arrow library. The main entry points are `CArrowArrayImporter.ImportArray`, `CArrowArrayExporter.ExportArray`, `CArrowArrayStreamImporter.ImportArrayStream`, and `CArrowArrayStreamExporter.ExportArrayStream` in the `Apache.Arrow.C` namespace. ([GH-33856](https://github.com/apache/arrow/issues/33856), [GH-33857](https://github.com/apache/arrow/issues/33857), [GH-36120](https://github.com/apache/arrow/issues/36120), and [GH-35809](https://github.com/apache/arrow/issues/35809)).

* `ArrowBuffer.BitmapBuilder.AppendRange(bool value, int length)` was added to improve performance of array concatenation ([GH-32605](https://github.com/apache/arrow/issues/32605)
```